src/frontends/onnx/tests/onnx_import.in.cpp

```diff
@@ -6736,6 +6736,25 @@ OPENVINO_TEST(${BACKEND_NAME}, onnx_model_reduce_min_20_boolean) {
     test_case.run();
 }
 
+OPENVINO_TEST(${BACKEND_NAME}, onnx_string_input) {
+    const auto model = convert_model("string_input.onnx");
+    auto test_case = test::TestCase(model);
+    test_case.add_input<std::string>({"strinpt1", "strinpt2"});
+    test_case.add_expected_output<int64_t>({2});
+    test_case.add_expected_output<std::string>({"strinpt1", "strinpt2"});
```

Comment:
```suggestion
    test_case.add_input<std::string>({"string1", "string2"});
    test_case.add_expected_output<int64_t>({2});
    test_case.add_expected_output<std::string>({"string1", "string2"});
```

Reply:
It was an original variant and it is equal to string_constant test, I want to be sure we use correct model for testing.